doc/nrf/libraries/bin/bt_ll_acs_nrf53/CHANGELOG.rst

```diff
@@ -9,6 +9,16 @@ Changelog
 
 Notable changes to this controller are documented in this file.
 
+Controller v3347
+****************
+
+The following changes have been introduced in the v3347 of the controller:
+
+Changes
+=======
+
+* ...........
```

Comment:
Was this forgotten?

Reply:
No Kristoffer said to open the PR so I could get input on this. Is there a list of changes somewhere I can copy in here?

Reply:
We may need a hand from @erikrobstad here so we can go through the relevant issues for this release.